zipline/pipeline/factors/factor.py

```diff
@@ -626,7 +632,9 @@ def rank(self, method='ordinal', ascending=True, mask=NotSpecified):
         return Rank(self, method=method, ascending=ascending, mask=mask)
 
     @expect_types(
-        target=Slice, correlation_length=int, mask=(Filter, NotSpecifiedType),
+        target=(FactorProxy, LoadableTerm, Slice),
```

Comment:
Note to fix: If the wrong type of term is given here, `expect_types` says something like "this parameter was expected to be of type FactorProxy or..." which is confusing since we actually mean `Factor`